src/put

```diff
@@ -273,24 +276,31 @@ def save(parser, context, args):
 	file_names = args.files
 
 	try:
-		file_ids = ','.join(get_file_ids(file_names))
+		file_ids = get_file_ids(file_names)
 	except ValueError as e:
 		print "Error:", e.message
 		exit(1)
-	response_json = api_call("files/zip?file_ids=" + file_ids)
-	zip_id = response_json["zip_id"]
-	print zip_id
-	while 1:
-		response_json = api_call("zips/"+str(zip_id))
-		if "url" in response_json:
-			if response_json["url"] != False:
-				break
-		time.sleep(1)
-	filename = str(os.getppid()) + ".zip"
-	download_command = config_json["downloader"].replace("$1", cache_folder).replace("$2", filename).replace("$3", response_json["url"])
-	os.system(download_command)
-	os.system("unzip " + cache_folder + "/" + filename)
-	os.system("rm -f " + cache_folder + "/" + filename)
+
+	if len(file_ids) == 1 and api_call("files/" + file_ids[0])["file"]["file_type"] != "FOLDER":
+		download_url = api_call("files/" + file_ids[0] + "/download", None, True)
+		download_command = config_json["downloader"].replace("$1", ".").replace("$2", file_names[0]).replace("$3", download_url)
```

Comment:
Now that constructing the download_command is done in multiple places (here and line 299), I would recommend pulling it out into a function.